docs/make.jl

```diff
@@ -1,7 +1,7 @@
 using Documenter, LazySets
 
 makedocs(
-#   doctest = false,  # uncomment locally for skipping doctests (saves time!)
```

Comment:
I would prefer to keep the `false` uncommented.
By default it is active, so ne need to write this explicitly.

Reply:
yes, this was a distraction, thanks

Reply:
well, actually i had written `true` in the right hand side :)

Reply:
You need to set it to `false`;)